client/my-sites/email/email-provider-features/list.js

```diff
@@ -11,36 +11,37 @@ const getEmailForwardingFeatures = () => {
 };
 
 const getGoogleAppLogos = () => {
+	const options = { textOnly: true };
```

Comment:
I know this is more verbose but I think `translateOptions` would better describe what that variable contains. Right now we have to read the rest of the code to understand what it is for.

Reply:
Also could we add a blank line below to separate this declaration from the rest of the code?